src/crawlee/parsel_crawler/_parsel_parser.py

```diff
@@ -0,0 +1,29 @@
+import asyncio
+from collections.abc import Iterable
+
+from parsel import Selector
+from typing_extensions import override
+
+from crawlee.http_clients import HttpResponse
+from crawlee.static_content_crawler._static_content_parser import StaticContentParser
+
+
+class ParselParser(StaticContentParser[Selector]):
+    """Parser for parsing http response using Parsel."""
+
+    @override
+    async def parse(self, response: HttpResponse) -> Selector:
+        return await asyncio.to_thread(lambda: Selector(body=response.read()))
+
+    @override
+    def is_matching_selector(self, parsed_content: Selector, selector: str) -> bool:
+        return parsed_content.type in ('html', 'xml') and parsed_content.css(selector).get() is not None
+
+    @override
+    def find_links(self, parsed_content: Selector, selector: str) -> Iterable[str]:
+        link: Selector
+        urls: list[str] = []
+        for link in parsed_content.css(selector):
+            if (url := link.xpath('@href').get()) is not None:
+                urls.append(url.strip())  # noqa: PERF401  #Mypy has problems using is not None for type inference in list comprehension.
```

Comment:
Seems that Mypy is OK with this: 
```suggestion
            url = link.xpath('@href').get()
            if url:
                urls.append(url.strip())
```